diff -up redis.conf.orig redis.conf
--- redis.conf.orig     2020-06-09 10:19:30.000000000 +0000
+++ redis.conf  2020-06-19 12:02:22.000000000 +0000
@@ -214,7 +214,7 @@ daemonize no
 #                        UPSTART_JOB or NOTIFY_SOCKET environment variables
 # Note: these supervision methods only signal "process is ready."
 #       They do not enable continuous liveness pings back to your supervisor.
-supervised no
+supervised systemd

 # If a pid file is specified, Redis writes it where specified at startup
 # and removes it at exit.
@@ -225,7 +225,7 @@ supervised no
 #
 # Creating a pid file is best effort: if Redis is not able to create it
 # nothing bad happens, the server will start and run normally.
-pidfile /var/run/redis_6379.pid
+pidfile /var/run/redis/redis.pid

 # Specify the server verbosity level.
 # This can be one of:
@@ -238,7 +238,7 @@ loglevel notice
 # Specify the log file name. Also the empty string can be used to force
 # Redis to log on the standard output. Note that if you use standard
 # output for logging but daemonize, logs will be sent to /dev/null
-logfile ""
+logfile /var/log/redis/redis.log

 # To enable logging to the system logger, just set 'syslog-enabled' to yes,
 # and optionally update the other syslog parameters to suit your needs.
@@ -343,7 +343,7 @@ rdb-del-sync-files no
 # The Append Only File will also be created inside this directory.
 #
 # Note that you must specify a directory here, not a file name.
-dir ./
+dir /var/lib/redis/

 ################################# REPLICATION #################################
