Index: asterisk-16.1.0/build_tools/get_documentation.py
===================================================================
--- asterisk-16.1.0.orig/build_tools/get_documentation.py
+++ asterisk-16.1.0/build_tools/get_documentation.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 # vin: sw=3 et:
 '''
 Copyright (C) 2012, Digium, Inc.
Index: asterisk-16.1.0/build_tools/post_process_documentation.py
===================================================================
--- asterisk-16.1.0.orig/build_tools/post_process_documentation.py
+++ asterisk-16.1.0/build_tools/post_process_documentation.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 # vin: sw=3 et:
 '''
 Copyright (C) 2012, Digium, Inc.
Index: asterisk-16.1.0/contrib/scripts/refcounter.py
===================================================================
--- asterisk-16.1.0.orig/contrib/scripts/refcounter.py
+++ asterisk-16.1.0/contrib/scripts/refcounter.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """Process a ref debug log
 
  This file will process a log file created by enabling
Index: asterisk-16.1.0/contrib/scripts/reflocks.py
===================================================================
--- asterisk-16.1.0.orig/contrib/scripts/reflocks.py
+++ asterisk-16.1.0/contrib/scripts/reflocks.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """Process a ref debug log for lock usage
 
  This file will process a log file created by Asterisk
Index: asterisk-16.1.0/contrib/scripts/refstats.py
===================================================================
--- asterisk-16.1.0.orig/contrib/scripts/refstats.py
+++ asterisk-16.1.0/contrib/scripts/refstats.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """Process a ref debug log for memory usage
 
  This will provide information about total and peak
Index: asterisk-16.1.0/contrib/scripts/sip_to_pjsip/sip_to_pjsip.py
===================================================================
--- asterisk-16.1.0.orig/contrib/scripts/sip_to_pjsip/sip_to_pjsip.py
+++ asterisk-16.1.0/contrib/scripts/sip_to_pjsip/sip_to_pjsip.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 
 from __future__ import print_function
 
Index: asterisk-16.1.0/contrib/scripts/sip_to_pjsip/sip_to_pjsql.py
===================================================================
--- asterisk-16.1.0.orig/contrib/scripts/sip_to_pjsip/sip_to_pjsql.py
+++ asterisk-16.1.0/contrib/scripts/sip_to_pjsip/sip_to_pjsql.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 
 from sip_to_pjsip import convert
 import sip_to_pjsip
Index: asterisk-16.1.0/contrib/scripts/spandspflow2pcap.py
===================================================================
--- asterisk-16.1.0.orig/contrib/scripts/spandspflow2pcap.py
+++ asterisk-16.1.0/contrib/scripts/spandspflow2pcap.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # vim: set ts=8 sw=4 sts=4 et ai tw=79:
 '''
 Usage: ./spandspflow2pcap.py SPANDSP_LOG SENDFAX_PCAP
Index: asterisk-16.1.0/contrib/scripts/voicemailpwcheck.py
===================================================================
--- asterisk-16.1.0.orig/contrib/scripts/voicemailpwcheck.py
+++ asterisk-16.1.0/contrib/scripts/voicemailpwcheck.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 ''' Sample externpasscheck script for use with voicemail.conf
 
 Copyright (C) 2010, Digium, Inc.
Index: asterisk-16.1.0/rest-api-templates/make_ari_stubs.py
===================================================================
--- asterisk-16.1.0.orig/rest-api-templates/make_ari_stubs.py
+++ asterisk-16.1.0/rest-api-templates/make_ari_stubs.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # Asterisk -- An open source telephony toolkit.
 #
 # Copyright (C) 2013, Digium, Inc.
Index: asterisk-16.1.0/contrib/scripts/ast_coredumper
===================================================================
--- asterisk-16.1.0.orig/contrib/scripts/ast_coredumper
+++ asterisk-16.1.0/contrib/scripts/ast_coredumper
@@ -251,7 +251,7 @@ EOF
 # For *BSD, the preferred gdb may be in /usr/local/bin so we
 # need to search for one that supports python.
 for g in $(which -a gdb) ; do
-	result=$($g --batch --ex "python print('hello')" 2>/dev/null || : )
+	result=$($g --batch --ex "python2 print('hello')" 2>/dev/null || : )
 	if [[ "$result" =~ ^hello$ ]] ; then
 		GDB=$g
 		break
